resources/views/layouts/admin-app.blade.php

```diff
@@ -22,18 +25,30 @@
         <img src="{{asset('img/logo.png')}}" alt="" class="logo">
         <nav id="nav-side">
           <ul class="admin-sidebar adminmenu-icon mt-5">
-            <li class="mb-5"><a href="#"><i class="fa-solid fa-users"></i></a></li>
+            <li class="mb-5"><a href="{{route('showUsers')}}"><i class="fa-solid fa-users"></i></a></li>
             <li class="mb-5"><a href="#"><i class="fa-sharp fa-solid fa-comment"></i></a></li>
-            <li class="mb-5"><a href=""><i class="fa-solid fa-inbox"></i></a></li>
-            <li class="mb-5"><a href=""><i class="fa-solid fa-calendar-check"></i></a></li>
+            <li class="mb-5"><a href="#"><i class="fa-solid fa-inbox"></i></a></li>
+            <li class="mb-5"><a href="#"><i class="fa-solid fa-calendar-check"></i></a></li>
           </ul>
         </nav>
     </header>
     <section class="top-bar">
         <h2 class="title mt-4 ms-5 mb-5">
           {{-- Users should be replaced for each pages--}}
           <span class="highlight"> Users </span> | <span>All Users</span>
-          <a class="logout float-end"><i class="fa-solid fa-arrow-right-from-bracket"></i></a>
+          {{-- <a class="dropdown-item" href="{{ route('logout') }}"
+          onclick="event.preventDefault();
+                        document.getElementById('logout-form').submit();">
+           {{ __('Logout') }}
+          </a> --}}
+
+
+          <a href="{{ route('logout') }}" class="logout float-end"  onclick="event.preventDefault();
```

Comment:
You can put this in <script></script> tag and put the function name in the onclick